content/applications/hr/referrals.rst

```diff
@@ -0,0 +1,703 @@
+=========
+Referrals
+=========
+
+Odoo's *Referrals* application is a centralized place where all information regarding referrals is
+housed, from points earned, coworkers hired, and rewards selected. The Referrals application
+integrates with the *Employees*, *Recruitment*, and *Website* applications, and must be installed in
+order for the Referrals application to function.
+
+There only configuration needed for the referrals application are the :ref:`rewards
+<referrals/rewards>`, everything else is pre-configured when Odoo Referrals is installed.
+
+All employees have access to the Referrals dashboard. Only managers and employees with specific
+access rights also have access to the :ref:`Reporting <referrals/reporting>` and Configurations
+menus.
+
+Onboarding
+==========
+
+When opening the *referrals* app for the first time, a pre-configured onboarding script appears.
+This is in the form of four slides, each explaining the different parts of the Referrals app. At the
+top of the dashboard, the following message is displayed throughout all the onboarding slides:
+`GATHER YOUR TEAM! Job Referral Program`. Behind this main message is an image, and beneath it some
+more explanatory text.
+
+There are four pre-configured onboarding slides. Each of the onboarding slides has a corresponding
+image and message that is displayed. After reading each message, click the :guilabel:`Next` button
+to advance to the next slide.
+
+The text that appears on each slide is as follows:
+
+#. `Oh no! Villains are lurking the city! Help us recruit a team of superheroes to save the day!`
+#. `Browse through open job positions, promote them on social media, or refer friends.`
+#. `Collect points and exchange them for awesome gifts in the shop.`
+#. `Compete against your colleagues to build the best justice league!`
```

Comment:
I always use too many images and have a bunch taken out by Zac, so I thought of doing the text since it's able to be edited, and this way, I wasn't relying on images (just explaining the logic for why I did it in this format).  I feel like keeping this in, since I do like to give all the info- plus I do have an example image below.